addons/survey/controllers/main.py

```diff
@@ -165,7 +165,10 @@ def survey_retry(self, survey_token, answer_token, **post):
             return werkzeug.utils.redirect("/")
 
         try:
-            retry_answer_sudo = survey_sudo._create_answer(user=request.env.user, partner=answer_sudo.partner_id, email=answer_sudo.email)
+            retry_answer_sudo = survey_sudo._create_answer(user=request.env.user, partner=answer_sudo.partner_id, email=answer_sudo.email, invite_token=answer_sudo.invite_token, **{
```

Comment:
The retry is the only case we create an new user_input with a given invite_token. Since it is given, attempt limit will be checked.

Reply:
Yes/Yes. Nothing to change here I think?